src/routes/MovieView.tsx

```diff
@@ -1,9 +1,63 @@
-const MovieView = () => {
+import './MovieView.css';
+
+export const MovieView = () => {
+  const movie = {
```

Comment:
Obviously this is temporary but the title of the movie will be the /:id in the url.
I suggest finding the movie by comparing the movie title from the JSON file to the params.id
consider using something like this:
`
const params = useParams();
const movie = movies.find((movie) => movie.title === params.id);
`

Reply:
Thank you for providing feedback! The issue at hand pertains solely to the creation of the Movieview page using temporary data placeholders. The connection to the JSON will be addressed in a subsequent issue. 👍